### PR TITLE
pt-br/ru: Replace {{bug}} macro calls with direct links

### PR DESCRIPTION
**Pull request by queengooborg**
This PR replaces all remaining calls to the `{{bug}}` macro with hard-coded URLs.  This removes the last remaining calls to the `{{bug}}` macro from all content.

### CODE REVIEW

**Comment by queengooborg** [comment]
The test failure is due to pre-existing issues in the files and is not a result of these changes.  I see that the test has now been made a required test to pass...

CC @caugner, we should probably fix all pre-existing issues in the localized content before making that test a required one.

**Comment by caugner** [comment]
> CC @caugner, we should probably fix all pre-existing issues in the localized content before making that test a required one.

Valid point, I removed `tests` from the required checks again. (cc @LeoMcA)

**Comment by queengooborg** [comment]
> Valid point, I removed `tests` from the required checks again.

It appears that it is still marked as a required test?

**Comment by caugner** [comment]
> It appears that it is still marked as a required test?

It looks like I didn't click on "Save changes". Now it's done.

File: files/pt-br/web/api/event/comparison_of_event_targets/index.md [hunk 1]
```diff
@@ -100,7 +100,7 @@ Há 5 tipos de _targets_ a se considerar:
         alguma outra razão senão o cruzamento de uma fronteira anônima, este
         evento será colocado no alvo antes que o redirecionamento ocorra. por
         exemplo, eventos do mouse são redirecionados à seus elementos pais
-        quando acontecem sobre nós de texto ({{ Bug("185889") }}), e neste
+        quando acontecem sobre nós de texto ([Firefox bug "185889"](https://bugzil.la/"185889")), e neste
```
Comment (caugner) [inline files/pt-br/web/api/event/comparison_of_event_targets/index.md:103]:
```suggestion
        quando acontecem sobre nós de texto ([Firefox bug 185889](https://bugzil.la/185889)), e neste
```

File: files/ru/conflicting/web/api/window/beforeunload_event/index.md [hunk 2]
```diff
@@ -31,7 +31,7 @@ window.onbeforeunload = function(e) {
...
+Когда это событие возвращает (или выставляет свойство returnValue) значение, отличное от null или undefined, пользователю предлагается подтвердить {{event("unload")}} страницы в {{domxref("window.prompt", "prompt")}} . В некоторых браузерах возвращаемое значение отображается в этом диалоговом окне. Начиная с Firefox 4, Chrome 51, Opera 38 и Safari 9.1 вместо возвращаемой строки будет показана общая строка, не контролируемая страницей. Например, Firefox отображает строку: "This page is asking you to confirm that you want to leave - data you have entered may not be saved.". Смотрите [Firefox bug "588292"](https://bugzil.la/"588292") и [Chrome Platform Status](https://www.chromestatus.com/feature/5349061406228480).
```
Comment (caugner) [inline files/ru/conflicting/web/api/window/beforeunload_event/index.md:34]:
```suggestion
Когда это событие возвращает (или выставляет свойство returnValue) значение, отличное от null или undefined, пользователю предлагается подтвердить {{event("unload")}} страницы в {{domxref("window.prompt", "prompt")}} . В некоторых браузерах возвращаемое значение отображается в этом диалоговом окне. Начиная с Firefox 4, Chrome 51, Opera 38 и Safari 9.1 вместо возвращаемой строки будет показана общая строка, не контролируемая страницей. Например, Firefox отображает строку: "This page is asking you to confirm that you want to leave - data you have entered may not be saved.". Смотрите [Firefox bug 588292](https://bugzil.la/588292) и [Chrome Platform Status](https://www.chromestatus.com/feature/5349061406228480).
```

File: files/ru/conflicting/web/css/_colon_focus-visible/index.md [hunk 3]
```diff
@@ -40,4 +40,4 @@ mybutton:-moz-focusring {
 
 ## Смотрите также
 
-- {{bug("418521")}}
+- [Firefox bug "418521"](https://bugzil.la/"418521")
```
Comment (caugner) [inline files/ru/conflicting/web/css/_colon_focus-visible/index.md:43]:
```suggestion
- [Firefox bug 418521](https://bugzil.la/418521)
```

File: files/ru/mozilla/firefox/releases/28/index.md [hunk 4]
```diff
@@ -13,8 +13,8 @@ Firefox 28 был выпущен 18 марта 2014 года. Список кл
 
 ### Инструменты разработчика
 
-- The {{domxref("console.exception")}} property has been added ({{bug("922214")}}).
-- The {{domxref("console.assert")}} property has been added ({{bug("760193")}}).
+- The {{domxref("console.exception")}} property has been added ([Firefox bug "922214"](https://bugzil.la/"922214")).
```
Comment (caugner) [inline files/ru/mozilla/firefox/releases/28/index.md:16]:
```suggestion
- The {{domxref("console.exception")}} property has been added ([Firefox bug 922214](https://bugzil.la/922214)).
```

File: files/ru/mozilla/firefox/releases/28/index.md [hunk 5]
```diff
@@ -13,8 +13,8 @@ Firefox 28 был выпущен 18 марта 2014 года. Список кл
...
+- The {{domxref("console.assert")}} property has been added ([Firefox bug "760193"](https://bugzil.la/"760193")).
```
Comment (caugner) [inline files/ru/mozilla/firefox/releases/28/index.md:17]:
```suggestion
- The {{domxref("console.assert")}} property has been added ([Firefox bug 760193](https://bugzil.la/760193)).
```

**Review by caugner — DISMISSED** [review]
Some quotes need to be stripped, my review comments don't include all occurrences.

**Review by josielrocha — APPROVED** [review]
Approve for `pt-BR`

**Review by sashasushko — APPROVED** [review]
Approve for Ru. Thanks ❤️

**Pull request merged**